packages/msk-insight-client/src/components/GeneFrequencyTableComponent.tsx

```diff
@@ -1,9 +1,8 @@
 import {action} from "mobx";
 import {observer} from "mobx-react";
+import {IGeneFrequencySummary} from "msk-insight-commons";
```

Comment:
coolest thing is that now we can refer to `msk-insight-commons`, even if it is not released on npm (or github or anywhere else)